presto-native-execution/presto_cpp/main/PrestoTask.cpp

```diff
@@ -64,6 +64,17 @@ protocol::RuntimeUnit toPrestoRuntimeUnit(RuntimeCounter::Unit unit) {
   }
 }
 
+// Presto has certain query stats logic depending on the operator names.
+// To leverage this logic we need to supply Presto's operator names.
+std::string toPrestoOperatorType(const std::string& operatorType) {
+  if (operatorType == "FilterProject") {
```

Comment:
This mapping is incorrect. Only FilterProject immediately on top of Scan becomes this. Also, with Aria enabled, there is no ScanFilterAndProjectOperator, only TableScan and ScanProject.

Reply:
Discussed offline.
Looks like we don't need to rename "FilterProject" to "ScanFilterAndProjectOperator" as in Native we never merge "FilterProject" with "TableScan" to get "ScanFilterAndProjectOperator" like Presto Java does.

Also meed to rename ExchangeOperator and MergeOperator.